examples/third_party/WORKSPACE.bazel

```diff
@@ -17,3 +17,5 @@ local_repository(
 load("//:repositories.bzl", "repositories")
 
 repositories()
+
+register_toolchains("//python:python_toolchain")
```

Comment:
This should not be needed now that the toolchain is registered in the `.bzl` file.

Reply:
oops, i meant to remove that. I've now removed ot